master/internal/kubernetes/events.go

```diff
@@ -77,3 +80,25 @@ func (e *eventListener) startEventListener(ctx *actor.Context) error {
 
 	return nil
 }
+
+func (e *eventListener) modMessage(msg *k8sV1.Event) {
+	replacements := map[string]string{
+		"Insufficient nvidia.com/gpu": "Waiting for resources. ",
+		"pod triggered scale-up":      "Job requires additional resources, scaling up cluster.",
+		"Successfully assigned":       "Cluster scaled up, pod resources allocated",
+		"skip schedule deleting pod":  "Deleting unscheduled pod",
+	}
+	resourceStatusText := " GPUs available, "
+
+	for k, v := range replacements {
+		matched, error := regexp.MatchString(k, msg.Message)
+		if error != nil {
+			break
+		} else if matched {
+			if v == "Waiting for resources. " {
```

Comment:
non-blocking: probably makes sense to define this string as a const since we are using it in multiple places.